mitodl/open-discussions
# Stabilized factory serialization

#### What are the relevant tickets?
#333 

#### What's this PR do?
This fixes the factory_data serialization so it doesn't error on subsequent runs. It also includes some ancillary changes (e.g. new fixtures in channels/views_test.py) because it was cherry-picked from another WIP branch.

#### How should this be manually tested?
Tests should pass. Delete the affected cassettes and regenerate them and then ensure a second test run passes.

#### What GIF best describes this PR or how it makes you feel?
![](https://media.giphy.com/media/gzKRbHzioNmzS/giphy.gif)
